### PR TITLE
Update scorecard.Rmd

### PR DESCRIPTION
**Pull request by MKensbock**
Corrections of ES DE before publication

### CODE REVIEW

**Review by MonikaFu — COMMENTED** [review]
See my comment.

File: inst/extdata/PA2024CH_de_exec_summary/scorecard.Rmd [hunk 1]
```diff
@@ -444,20 +444,20 @@ Die Antworten, die für jede Frage berücksichtigt wurden, sind im Folgenden auf
 
 ### Glaubwürdige Klimaverantwortung
 
-_**Umsetzung:** weicht von den strikten Anforderungen für die Swiss Climate Scores ab (siehe unten)._
+_**Umsetzung:** "eicht von den strikten Anforderungen für die Swiss Climate Scores ab (siehe unten)._
```
[the remark this answers is not in the document]
Comment (MKensbock) [reply inst/extdata/PA2024CH_de_exec_summary/scorecard.Rmd:447]: No, that should be a W - thank you!

[the remark this answers is not in the document]
Comment (jdhoffa) [reply inst/extdata/PA2024CH_de_exec_summary/scorecard.Rmd:447]:
```suggestion
_**Umsetzung:** Weicht von den strikten Anforderungen für die Swiss Climate Scores ab (siehe unten)._
```

[the remark this answers is not in the document]
Comment (MonikaFu) [reply inst/extdata/PA2024CH_de_exec_summary/scorecard.Rmd:447]: I believe Matze alredy changed it.

**Review by MonikaFu — APPROVED** [review]
LGTM

File: inst/extdata/PA2024CH_de_exec_summary/scorecard.Rmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (MonikaFu) [reply inst/extdata/PA2024CH_de_exec_summary/scorecard.Rmd:447]: I was waiting for the checks to pass.

**Pull request merged**